CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Guide to Contributions
+
+## Introduction
+
+Hello and welcome to JABACAT! We're happy that you want to contribute to our
+projects. However, to maintain order, we ask that you contribute according to
+these guidelines to keep with our current practices.
+
+# General Guidelines For All Projects
+
+## GitHub branches and forks
+
+We ask that you not push any code directly to our main repositories, unless it
```

Comment:
Maybe instead of main, it could be upstream "jabacat/{project}" to not confuse with just the main branch. Anything on the upstream version of the repository

Reply:
How about "to the repositories directly on `https://github.com/jabacat/`"?

Reply:
People might not realize that means pushing to upstream, especially if they have it set as origin. Hm, this is hard to convey.